webpack/ForemanInventoryUpload/SubscriptionsPageExtension/SubscriptionsPageExtensionActions.js

```diff
@@ -0,0 +1,44 @@
+import API from 'foremanReact/API';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { foremanUrl } from '../../ForemanRhCloudHelpers';
+import { inventoryUrl } from '../ForemanInventoryHelpers';
+
+export const subscriptionCountListener = (items, unsubscribe) => {
+  if (items === undefined || items === 0) {
+    return;
+  }
+
+  fetchInventoryAutoUploadSetting(showToast, unsubscribe);
+};
+
+const showToast = (autoUploadEnabled, unsubscribe) => {
```

Comment:
Can you move `showToast` to the scope of `subscriptionCountListener`?
then you don't need to pass `unsubscribe` as a parameter because it's already exist in the scope.